dags/sentinel-2_data_transfer.py

```diff
@@ -151,15 +158,19 @@ def start_transfer(message):
         raise ValueError(
             f"{key} does not exist in the {SRC_BUCKET_NAME} bucket"
         )
-
-    urls = [s3_filepath]
+    
+    try:
+        s3_hook_oregon.load_string(string_data=updated_stac, key=key, bucket_name=DEST_BUCKET_NAME)
```

Comment:
The destination bucket is in cape town, will this work?

Reply:
Nope, this was from test environment.  It's changed to a Cape Town connection.